CMakeLists.txt

```diff
@@ -12,19 +12,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 cmake_minimum_required(VERSION 3.17.2)
 
-project(VULKAN_UTILITY_LIBRARIES LANGUAGES CXX)
+project(VUL LANGUAGES CXX)
```

Comment:
NOTE: Changing the project name to be brief also makes accessing the variables generated by the project command simpler.

EX:

`VUL_SOURCE_DIR` vs `VULKAN_UTILITY_LIBRARIES_SOURCE_DIR`

Reply:
ok